R/toolGetMapping.R

```diff
@@ -31,6 +31,16 @@ toolGetMapping <- function(name, type = NULL, where = NULL,
                            returnPathOnly = FALSE, activecalc = NULL) {
 
   setWrapperInactive("wrapperChecks")
+  if (isWrapperActive("wrapperChecks")) {
```

Comment:
Why are you testing this condition? As this wrapper is being activated in the line before, your new code portion is never executed.

Reply:
Once you remove this line, a lot of tests fail. 

As a firsts step, you have to deal with the instances where the tests call madrat functions like `madrat::regionscode` which currently call `toolGetMapping` without a where argument and therefore now produce the new warning. Here, I am not sure what the right approach is: either add the `where` argument or enhance the check to not throw the warning when called from within madrat functions. @pfuehrlich-pik what do you think?

There might be other reasons why the test fail, but we can look into them once this reason has been removed.

Reply:
Passing this question on to @tscheypidi

Reply:
the `isWrapperActive` should exactly achieve that the checks are only active outside of madrat functions. What happens if the checks are being performed before the `wrapperChecks`-wrapper is being deactived?